Components/inputs.html

```diff
@@ -97,16 +110,104 @@ <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
-              style="
-                width: 100%;
-                height: 682px;
-                border: 0;
-                transform: scale(1);
-                overflow: hidden;
-              "
-              sandbox="allow-scripts allow-same-origin"
-            >
+            src="https://carbon.now.sh/embed/UsIToXjaUTzlzXswlO0p"
+            style="width: 100%; height: 929px; border:0; transform: scale(1); overflow:hidden;"
+            sandbox="allow-scripts allow-same-origin">
+          </iframe>
+            </iframe>
+          </div>
+        </div>
+
+
+        <!-- ** Form Input ** -->
+
+        <div class="exampleCont">
+          <h3 class="exampleText">Form Input</h3>
+
+          <div class="flexCont">
+
+            <!-- Required Text Input-->
+
+            <div class="input-cont">
+              <input
+                type="text"
+                name="fullname"
+                class="simpleText-input"
+                placeholder="Required"
+                required
+              />
+            </div>
+
+            <!-- Disabled input-->
+
+            <div class="input-cont">
+              <input
+                type="text"
+                name="fullname"
+                class="simpleText-input disabled-input"
+                placeholder="Disabled"
+                disabled
+              />
+            </div>
+
+            <!--Password Input-->
+
+            <div class="input-cont">
+              <input
+                type="password"
+                name="fullname"
+                class="simpleText-input"
+                placeholder="password"
+                required
+              />
+            </div>
+
+            <!--Email Input-->
+
+            <div class="input-cont">
+              <input
+                type="email"
+                name="fullname"
+                class="simpleText-input"
+                placeholder="Email"
+                required
+              />
+            </div>
+
+            <!--Number input-->
+
+            <div class="input-cont">
+              <input
+                type="number"
+                name="fullname"
+                class="simpleText-input"
+                placeholder="Number"
+                required
+              />
+            </div>
+
+            <!-- Date Input -->
+
+            <div class="input-cont">
+              <input
+                type="date"
+                name="fullname"
+                class="simpleText-input"
+                placeholder="Date"
+                required
+              />
+            </div>
+           
+          </div>
+
+          <h3 class="showCodeToggle" onclick="showCodeToggle()"></h3>
+
+          <div class="code hideCode">
+           <iframe
+  src="https://carbon.now.sh/embed/ZGdR3Gl4CJm9CtMrHbxD"
+  style="width: 100%; height: 1518px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
```

Comment:
This showing list component snippits
![Screenshot (58)](https://user-images.githubusercontent.com/42497931/112652615-99f0a400-8e73-11eb-895c-47bc1a8efac0.png)

Reply:
Fixed